chesster.js

```diff
@@ -645,6 +648,97 @@ controller.hears([
     });
 });
 
+controller.hears([
+    /pairing(?: @?([^\s]+))?/
+], [
+    'direct_mention',
+    'direct_message'
+], function(bot, message) {
+    bot_exception_handler(bot, message, function() {
+        bot.startPrivateConversation(message, function (response, convo) {
+            // The user is either a string or an id
+            var nameOrId = message.match[1];
+            var requesting_player = users.getByNameOrID(message.user);
+
+            // The name or Id was provided, so parse it out
+            var player = null;
```

Comment:
it is not necessary, I think it's left over cruft from my development.  I should really pay closer attention to that.  Thanks for pointing it out.  I will remove it and resubmit!